### ISSUE TITLE
no-navigation-without-resolve should accept mixed ternaries

### ISSUE THREAD

**Issue by marekdedic** [sveltejs/eslint-plugin-svelte#1389]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

latest

### What version of `eslint-plugin-svelte` are you using?

main

### What did you do?

```svelte
<a href={foo ? "https://example.com" : resolve("/")}
```

### What did you expect to happen?

No error

### What actually happened?

Error

### Link to **GitHub Repo** with Minimal Reproducible Example

N/A

### Additional comments

Will look at this eventually, but this will probably require a big rewrite of some parts of the rule

**Comment by tv42** [comment]
Similar use case, allow unsetting the attribute:

```svelte
<a href={disabled ? undefined : resolve("/")}
```
